Sources/Controllers/2.0/OAMapSettingsMainScreen.mm

```diff
@@ -232,11 +232,11 @@ - (void) setupView
         _filteredTopLevelParams = [[_styleSettings getParameters:@""] filteredArrayUsingPredicate:[NSPredicate predicateWithFormat:@"(_name != %@) AND (_name != %@) AND (_name != %@)", kContourLinesDensity, kContourLinesWidth, kContourLinesColorScheme]];
         NSMutableArray *categoriesList = [NSMutableArray array];
         NSString *modeStr;
-        if ([_settings.settingAppMode get] == APPEARANCE_MODE_DAY)
+        if ([_settings.dayNightMode get] == APPEARANCE_MODE_DAY)
             modeStr = OALocalizedString(@"map_settings_day");
-        else if ([_settings.settingAppMode get] == APPEARANCE_MODE_NIGHT)
+        else if ([_settings.dayNightMode get] == APPEARANCE_MODE_NIGHT)
             modeStr = OALocalizedString(@"map_settings_night");
-        else if ([_settings.settingAppMode get] == APPEARANCE_MODE_AUTO)
+        else if ([_settings.dayNightMode get] == APPEARANCE_MODE_AUTO)
```

Comment:
if constants start with APPEARANCE, settings should start with appearance as well. I.e. **appearanceMode**